.pullapprove.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+
+requirements:
+  signed_off_by:
+    required: false
+
+always_pending:
+  title_regex: 'WIP'
```

Comment:
this is regular expression, so I guess it is case sensitive, but you can write it as `WIP|wip`, I don't know if they use the insensitive mode when matching using regexp. `i` should turn it on, see [this question](http://stackoverflow.com/questions/9655164/regex-case-sensitive)